core/domain/user_id_migration.py

```diff
@@ -47,11 +47,57 @@ class MissingUserException(Exception):
     pass
 
 
+class CreateNewUsersMigrationJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for creating new UserSettingsModels with new user ids set.
+    This migration doesn't handle the replacement of old user ids in the other
+    models, this is done by the UserIdMigrationJob.
```

Comment:
Here and in other jobs: if two jobs need to be run one after the other to ensure consistent results, add this information in the docstrings of the jobs (to be clear that this job isn't truly independent).

Reply:
Done